src/orderbook.js

```diff
@@ -1204,20 +1343,24 @@ OrderBook.prototype.computeAutobridgedOffers = function() {
  */
 
 OrderBook.prototype.mergeDirectAndAutobridgedBooks = function() {
-  const self = this;
+
+  if (this._destroyed) {
+    return;
+  }
 
   if (_.isEmpty(this._offers) && _.isEmpty(this._offersAutobridged)) {
+    if (this._synchronized && this._gotOffersFromLegOne &&
+      this._gotOffersFromLegTwo) {
+      // emit empty model to indicate to listeners that we've got offers,
+      // just there was no one
+      this.emit('model', []);
+    }
     return;
   }
 
   this._mergedOffers = this._offers
     .concat(this._offersAutobridged)
-    .sort(function(a, b) {
-      const aQuality = OrderBookUtils.getOfferQuality(a, self._currencyGets);
-      const bQuality = OrderBookUtils.getOfferQuality(b, self._currencyGets);
-
-      return aQuality.compareTo(bQuality);
-    });
+    .sort(_sortOffers.bind(this));
```

Comment:
from earlier processing, each order has a `quality` field as a numerical string - can we or should we just use that for the sort rather than creating Amount objects? 

```
function (a, b) {
  return Number(a.quality) - Number(b.quality);
}
```